tomviz/python/tomviz/state/_schemata.py

```diff
@@ -189,6 +189,7 @@ class Meta:
 class ReaderSchema(Schema):
     fileNames = fields.List(fields.String)
     name = fields.String()
+    subsampleSettings = fields.Dict(values=fields.List(fields.Int()))
```

Comment:
I think here we can add a field definition for SubSampleSettings, that explicitly defines the fields expected?

Reply:
Done.